.changeset/popular-hornets-promise.md

```diff
@@ -0,0 +1,9 @@
+---
+"@hashicorp/design-system-components": patch
```

Comment:
[question] I think we've discussed this before so forgive my poor memory, but since these are major changes upstream would a `minor` be a better way to communicate these changes?

Reply:
I know there is no standard in the industry around this, unfortunately. My thinking is that if dependency bump (be that a major) materializes in a feature for our consumers (let's say tippy allows a new option that cascades in our component) then it's a feature. In all other instances it's a patch. Changesets and Lerna operate similarly. For example when we bumped `@hashicorp/ember-flight-icons` to `3.0.0` this was forwarded by changesets into a patch for `@hashicorp/design-system-components`. I'm happy to review this as a team if we feel differently about it. I'm aware that we might enforce these majors to our consumers, and that's not ideal.

Reply:
> Changesets and Lerna operate similarly

Good point and good context. I think your explanation makes sense to keep it as you have it here.